### DISCUSSION TITLE
How to turn off pyright and turn on pylsp - config.example.lua misleading?

### DISCUSSION THREAD

**Discussion by dstromberg** [LunarVim/LunarVim#3764] [General]
Hello people.

I'm attempting to turn off pyright and turn on pylsp with ruff.

I've set up lunarvim with:
`LV_BRANCH='release-1.2/neovim-0.8' bash <(curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/fc6873809934917b470bff1b072171879899a36b/utils/installer/install.sh)
`
I found an example of turning off a LS and turning one on as well in ~/.local/share/lunarvim/lvim/utils/installer/config.example.lua
...but it might be a little off, because it doesn't appear to be working for me.  I've tried:

```
-- Turn off pyright, turn on pylsp - for speed
vim.list_extend(lvim.lsp.automatic_configuration.skipped_servers, { "pyright" })
lvim.lsp.automatic_configuration.skipped_servers = vim.tbl_filter(function(server)
  return server ~= "pylsp"
end, lvim.lsp.automatic_configuration.skipped_servers)

```
About the above, sumneko_lua says:
`vim.list_extend(lvim.lsp.automatic_configuration.skipped_servers, { "pyright" })     ■ This function requires 4 argument(s) but instead it is receiving 2.`

My hope is to turn off pyright and turn on pylsp with ruff for speed, because pyright is taking about one second for every little change in normal mode.

In fact, just using ":LspUninstall pyright" followed immediately by ":LspInstall pylsp" fails - there are no errors, but pyright kicks right back in.

In fact, just ":LspUninstall" followed by an lvim restart brings me right back to pyright.

I don't really want to use :Lsp* though - I'd much rather do this automatically, from a shell script.  I just used :Lsp* above to show a little more info about what I'm seeing.

What do I need to do to eliminate the tenacious pyright and install pylsp?

Thanks!

**Comment by LostInTheLogs** [comment]
see https://www.lunarvim.org/docs/master/configuration/language-features/language-servers#changing-the-default-server

**Reply by dstromberg** [reply]
Thanks, that helped.  I needed the :LvimCacheReset
